code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -47,6 +47,7 @@
 		// If we're not exploding, set the alert level back to normal
 		set_safety()
 	GLOB.nuke_list -= src
+	explosion(src, 50, 70, 80, 100, TRUE, TRUE) //SKYRAT EDIT ADDITION
```

Comment:
Just a heads up. If the blob eats the nuke, it causes this detonation and makes big boom boom.